src/framework/theme/components/popover/popover.directive.ts

```diff
@@ -77,7 +77,7 @@ import { NbPlacementHelper } from './helpers/placement.helper';
  * - Hover mode works like hint mode with one exception - when the user moves mouse from host element to
  * the container element popover will not be hidden.
  *
- * @example(popover/popover-modes.component)
+ * @stacked-example(Available modes, popover/popover-modes.component.html)
```

Comment:
why not rename everywhere?

Reply:
Because we'll have to add names for all examples and I think it would slightly convenient to rename in that time.

Reply:
done